azure-pipelines.yml

```diff
@@ -57,16 +57,36 @@ jobs:
       toolMajorVersion: V2
       suppressionsFile: './scripts/ci/credscan/CredScanSuppressions.json'
       toolVersionV2: '2.1.17'
+
   - task: ms-codeanalysis.vss-microsoft-security-code-analysis-devops.build-task-postanalysis.PostAnalysis@1
     displayName: 'Post Analysis'
     inputs:
       AllTools: false
       BinSkim: false
       CredScan: true
+      PoliCheck: false
       RoslynAnalyzers: false
       TSLint: false
       ToolLogsNotFoundAction: 'Standard'
 
+- job: PolicyCheck
+  displayName: "Policy Check"
+  pool:
+    name: ${{ variables.windows_pool }}
+  steps:
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+    displayName: 'Run Policy Check'
+    inputs:
+      targetType: F
+      result: PoliCheckResult.xml
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
```

Comment:
We may have some documents for how this YAML job is generated.

Reply:
I will use the classic editor to create a pipeline without YAML.
In this way you can check the meaning of each parameter directly on the UI.
After testing, you can use the **View YAML** function to automatically convert to yaml format.
![image](https://user-images.githubusercontent.com/18628534/202380184-4612bc6f-28e9-4973-93dc-b8c40e4be2f7.png)
![image](https://user-images.githubusercontent.com/18628534/202380258-2a56f326-1eea-4adb-ae83-9f3523c2cc06.png)